lib/index.ts

```diff
@@ -71,7 +71,7 @@ class BlueLinky extends EventEmitter {
       this.emit('ready', this.vehicles);
       return response;
     } catch (error) {
-      throw new Error(error.message);
+      return error;
```

Comment:
What's the thought behind returning an error rather than throwing it?

Reply:
Not having unhandled rejects in promises, as it provides no log info